src/common/config_parser.h

```diff
@@ -51,14 +51,14 @@ class ConfigParser {
   YAML::Node getConfig() const;
   std::vector<DeviceId> getDevices();
 
-  // Check if the config has a boolean option for the given key and it is
-  // enabled
-  bool has(const std::string& key) const;
-
 private:
   cli::mode mode_;
   YAML::Node config_;
 
+  // Check if the config has a boolean option for the given key and it is
+  // enabled
+  bool hasBool(const std::string& key) const;
```

Comment:
Not a fan of the name "hasBool" we have "has" everywhere else

Reply:
What's the use case here? And what about a more generic has<T> ?

Reply:
From the several instances, I think this function is not actually needed; instead, all calls should be rewritten as:

    get<bool>("key", false)

Reply:
Changed this to get, but considering to remove this and go back to YAML function. We use YAML for everything else here, the function is a bit obfuscating.